### DISCUSSION TITLE
Famous Keyboard & Synth Sounds of the 80s RECREATED!

### DISCUSSION THREAD

**Discussion by probonopd** [probonopd/MiniDexed#483] [General]
[Famous Keyboard & Synth Sounds of the 80s RECREATED!](https://www.youtube.com/watch?v=ccX4SEv-Hl8) by 
JimDaneker.

These are the sounds that got me into this hobby.

I can smell a challenge for @Banana71 here ;-)

**Comment by diyelectromusic** [comment]
Oh, and that follow up video... :)

https://www.youtube.com/watch?v=bkQ9z6Q77Z8&t=0s

**Reply by probonopd** [reply]
I like "this montrosity called Chewie, We're Home" at 4:40... 
This is 8 layers of TX816, but each with a chorus of its own...

https://youtu.be/bkQ9z6Q77Z8?t=280

**Comment by Banana71** [comment]
Whitney Houston - Greatest Love of All
Based on the sound DX7IIVoice64 - 30 HardRoads 
add. Bells and layer. If you want to be closer to the original, you should make TG5&6 quieter (Bells) 

[000015_mDx-Dreams.zip](https://github.com/probonopd/MiniDexed/files/11285358/000015_mDx-Dreams.zip)
